WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -167,23 +167,24 @@ final class EditableOrderViewModel: ObservableObject {
 
     /// View model for the product list
     ///
-    lazy var addProductViewModel = {
+    var handleProductsViewModel: ProductSelectorViewModel {
```

Comment:
Can we add a more specific name than `handleProductsViewModel`?

Reply:
Definitely! Now that not only adds products, it seemed appropriate to just name it as its definition, so I renamed it to `productSelectorViewModel` here https://github.com/woocommerce/woocommerce-ios/pull/8943/commits/aed18f77fe9bcc47cb0e40f8d42a76ba2a012e4a